setup.py

```diff
@@ -58,16 +56,13 @@ def run(self):
 
 
 requirements = [
-    "Django~=2.2.3",
-    "galaxy-importer==0.2.13",
-    "pulpcore>=3.7,<3.9",
-    "pulp-ansible==0.5.5",
-    "django-prometheus>=2.0.0",
-    "drf-spectacular",
+    "pulp-ansible",
+    "django-prometheus",
 ]
 
 package_name = os.environ.get("GALAXY_NG_ALTERNATE_NAME", "galaxy-ng")
-version = os.environ.get("ALTERNATE_VERSION", __version__)
+# version = os.environ.get("ALTERNATE_VERSION", __version__)
+version = "4.3.0.dev"
```

Comment:
version is defined on `__init__.py` and then repeated here again, I think we should have a single place where version is defined, in this case here we can get back to the approach of reading it `statically` from `__init__.py`